### ISSUE TITLE
Variables supporting

### ISSUE THREAD

**Issue by turbotankist** [VictoriaMetrics/victorialogs-datasource#5]
Hey!
Thank you for the datasource pluging.

I wanna try it, but seems like it doen't support grafana variables yet.
I created the variable 
![image](https://github.com/VictoriaMetrics/victorialogs-datasource/assets/9462236/bf7c8fa2-52de-4ec1-b58a-722f5be6b5a9)
then added the query `kubernetes.pod_namespace: $namespace`
But the query insector shows
```
datasource:Object
expr:"_time:[2024-02-19T02:21:09.656Z, 2024-02-19T08:21:09.656Z] AND kubernetes.pod_namespace: $namespace"
queryType:"range"
refId:"A"
```

**Comment by dmitryk-dk** [comment]
Hi @turbotankist ! Thank you for the issue. We will take a look.

**Comment by dmitryk-dk** [comment]
Hi @turbotankist ! This bug has been fixed in [v0.2.0 release](https://github.com/VictoriaMetrics/victorialogs-datasource/releases/tag/v0.2.0). Closing the issue as fixed then.
